rtc/configuration.md

```diff
@@ -0,0 +1,371 @@
+
+### Example Using the document details provider to return static document details
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_document_details_provider: () => {
+    return Promise.resolve({ documentId: "your-document-id" })
+  }
+})
+```
+
+### Example Using the document details provider to return document details from your server
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_document_details_provider: () => {
+    return fetch('http://yourserver/getDocumentId', {
+      method: 'POST'
+    })
+    .then(documentId => ({ documentId });
+  }
+})
+```
+
+## `rtc_encryption_provider`
+
+The RTC plugin requires a generated key to encrypt messages for end-to-end encryption. This key not sent to the {{site.cloudname}}, preventing the {{site.productname}} RTC service from reading the editor content. The encryption key is used by the user's browser to encrypt and decrypt the editor content between users.
+
+To generate a unique encryption key, create a function that accepts the following arguments:
+* The document ID
+* The session ID
+* A key hint (only required for existing sessions)
+
+Secure ways to generate an encryption key include:
+
+* Generate and store a random key for each new document session in your database.
+* Generate single random key per document, and salt it _on your server_ with the session ID to provide a different key per session.
+* Store a global list of keys for your application, and use the document ID and session ID to salt the current key _on your server_ to produce a key unique to the document session. You should return a key hint to identify the key in case you rotate to a new key later.
+
+**Type:** `Function`
+
+**Required:** yes
+
+### Input fields for `rtc_encryption_provider`
+
+| Field | Type | Description |
+|-------|:----:|-------------|
+| `documentId` | string | Unique public document ID that is used when sharing the document. |
+| `sessionId` | integer | Session ID generated by the server (0 ≤ sessionId < 2<sup>31</sup>). |
+| `keyHint` | string/undefined | Key hint (such as a key ID) provided by the client which opened the session (only required if there is an existing session). |
+
+### Return fields for `rtc_encryption_provider`
+
+| Field | Type | Description |
+|-------|:----:|-------------|
+| `key` | string | Encryption key that is used to locally encrypt operations. This key needs to be the same for all connecting clients. |
+| `keyHint` | string/undefined | Optional key hint to provide to later clients to aid in key selection. This can be a key thumbprint, ID or other non-sensitive identifier that will help select the key like a timestamp. It is only recorded when opening a new session. (unicode, max 256 characters) |
```

Comment:
```suggestion
| `keyHint` | string or undefined | Optional key hint to provide to clients that connect later, to aid in key selection. This can be a key thumbprint, ID or other non-sensitive identifier that will help select the key like a timestamp. It is only recorded when opening a new session. (unicode, max 256 characters) |
```